extensions/pyRevitTools.extension/pyRevit.tab/Selection.panel/memo.stack3/Paste State.pushbutton/script.py

```diff
@@ -401,17 +405,19 @@ def set_tansform_matrix(selvp, selview):
         f = open(datafile, 'r')
         cloops_data = pickle.load(f)
         f.close()
+        selected_elements = revit.get_selection().elements
         with revit.Transaction('Paste Crop Region'):
-            revit.activeview.CropBoxVisible = True
```

Comment:
I removed the line, that enables CropBox for updates view. I find it unnecessary to change it after applying a new Crop, especially when more than one view can be updated.
If the line was added because of Revit API specific, maybe it makes sense to set it back to `False` if Crop was hidden.